openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -122,35 +162,19 @@ def _recurring_nudge_schedules_for_hour(target_hour, org_list, exclude_orgs=Fals
     if "read_replica" in settings.DATABASES:
         schedules = schedules.using("read_replica")
 
-    dashboard_relative_url = reverse('dashboard')
+    return users, schedules
 
-    for (user, user_schedules) in groupby(schedules, lambda s: s.enrollment.user):
-        user_schedules = list(user_schedules)
-        course_id_strs = [str(schedule.enrollment.course_id) for schedule in user_schedules]
 
-        def absolute_url(relative_path):
-            return u'{}{}'.format(settings.LMS_ROOT_URL, urlquote(relative_path))
+def add_upsell_button_to_email_template(upgrade_data, template_context):
+    is_unverified = False
```

Comment:
nit: I think I would prefer "show_upsell", "is_unverified" feels like it could result in double negatives in conditionals (it doesn't in this case, but could in the future). Also we don't really use the term "unverified".